nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,76 @@
+const { request, response } = require("express");
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/meals.json");
+
+//Respond with the json for the meals with the corresponding id
+router.get("/:id", async (request, response) => {
+  try {
+    const id = parseInt(request.params.id);
+    console.log(id);
+    const mealsId = meals.find((meal) => meal.id === id);
+    if (mealsId) {
+      response.json(mealsId);
+    } else {
+      return response.status(400).send("This ID Not matching any meal ids ");
+    }
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/", async (request, response) => {
```

Comment:
good work! I have a use case if you want to work on a bit more and make it more complete:
What if I want to use more than one query parameter, for instance have both title and maxPrice? Currently you are using response.json directly, so I will get results based on only the first parameter. What could be done differently?
For instance, if make this request:
```
http://localhost:3000/api/meals?maxPrice=1000&limit=1
```
I still get all the results